rbs/AssertionsRewriter.cc

```diff
@@ -732,6 +732,12 @@ unique_ptr<parser::Node> AssertionsRewriter::rewriteNode(unique_ptr<parser::Node
             splat->var = rewriteNode(move(splat->var));
             result = move(node);
         },
+        [&](parser::Super *super_) {
+            for (auto &expr : super_->args) {
+                expr = rewriteNode(move(expr));
```

Comment:
Why does this line not have a `maybeInsertCast` here, while the similar line in `Array` for loop does?